dvc/command/repro.py

```diff
@@ -48,6 +48,7 @@ def run(self):
                 if self.args.metrics:
                     metrics = self.repo.metrics.show()
                     show_metrics(metrics)
+                logger.info("Don't forget to execute 'dvc push'")
```

Comment:
@efiop do we want to check if `--no-commit` is provide and alter the message a bit? Can we do it here?

@DmitriyVahrushev thanks! I think it would be better to do the message a bit more formal - `Run dvc push if you want to save your data to remote storage`

@jorgeorpinel could you please review the message? :)

Reply:
@shcheklein totally, `if not self.args.no_commit:` wouldn't hurt there, for sure.

Reply:
> please review the message

How about `"To save the updates to remote storage, run `dvc push`."` ?